### ISSUE TITLE
How can I get a list windows across all sessions sorted by 'most recent'?

### ISSUE THREAD

**Issue by ntwk** [tmux/tmux#2663]
Is there a way to output a list of all windows on the server, sorted by 'most recent'?  In this case, I'd like 'most recent' to be mean 'most recently displayed' in a manner similar to how the `session_stack` variable works, but for all sessions.

I've experimented with the following:

`tmux list-windows -a`

While this lists all windows on the server, it does not specify a sort order.

I've also tried:

`tmux list-windows -a -F '#{window_activity}:#{window_id}:#W' | sort -r`

However, this lists windows sorted by most recent activity where 'activity' includes output to the window.  If a window hasn't been visited recently but it frequently receives output it will appear at the top of the list which isn't quite what I want.

To summarize:

I'm trying to get a list of all windows on the server in the order in which they were last visited.

My environment is as follows:

- Platform: Linux 5.11.2
- Tmux version: 3.1c
- TERM inside tmux: tmux-256color
- TERM outside tmux: rxvt-unicode-256color

**Comment by nicm** [comment]
You can look at session_stack for each session but tmux does not track time-of-last-use rather than time-of-last-activity so there is no way to do this across sessions.

**Comment by ntwk** [comment]
Thanks for looking at this.  At least I know I'm not missing anything here.  I'll see if I can make do with the `window-activity` variable.
